### PR TITLE
More docs design changes

### PR DESCRIPTION
**Pull request by mdo**
- Updated hero section
- New clipboard icon button instead of text
- Temporary full bootstrap minified CSS
- Syntax tweaks for shell

Preview: <https://deploy-preview-528--bootstrap-icons.netlify.app/>

TODO:

- [ ]  Move some patches to separate PRs

### CODE REVIEW

**Comment by mdo** [comment]
@XhmikosR This is intended to go with v1.2.0 as well. Brings the icons docs more up to speed with v5's docs.

**Comment by XhmikosR** [comment]
I'll have a close look tomorrow. :)

I still don't quite get why we include a purged Bootstrap instead of vanilla :P I mean, sure it's slimmer, but there are no scripts for others to reproduce the file.

We could just build Bootstrap ourselves with Hugo and just not include all components?

**Comment by mdo** [comment]
> I still don't quite get why we include a purged Bootstrap instead of vanilla :P I mean, sure it's slimmer, but there are no scripts for others to reproduce the file.
> 
> We could just build Bootstrap ourselves with Hugo and just not include all components?

Yeah, I went down that path of even including the `dist` CSS via npm, but opted to leave it as-is for now. We can do that separately whenever we get a chance.

File: docs/layouts/partials/hero.html [hunk 1]
```diff
@@ -1,34 +1,42 @@
...
+      <div class="col-6 mx-auto col-lg-5 mx-lg-0">
+        <img srcset="/assets/img/icons-hero.png,
+                     /assets/img/icons-hero@2x.png 2x"
+                src="/assets/img/icons-hero.png"
+                class="img-fluid my-3 mx-auto"
+                alt="Bootstrap Icons" width="450" height="340" loading="lazy">
       </div>
-      <div class="col-md-3 col-lg-4">
+      <div class="col-lg-7">
+        <a class="d-inline-block py-1 px-3 mb-4 text-dark text-decoration-none rounded-pill hero-notice" href="#">
```
Comment (XhmikosR) [inline docs/layouts/partials/hero.html:12]: This goes nowhere right now?

Comment (mdo) [reply docs/layouts/partials/hero.html:12]: Yeah, I'm thinking it'll link to the blog post, but won't update the URL until the v1.2.0 prep PR is more ready along with a blog post PR.

File: docs/content/_index.md [hunk 2]
```diff
@@ -5,21 +5,21 @@
 
 Bootstrap Icons are published to npm, but they can also be manually downloaded if needed.
 
-<div class="row my-4">
+<div class="row my-4 bd-clipboard-simplified">
```
Comment (XhmikosR) [inline docs/content/_index.md:8]: `bd-clipboard-simplified` seems unused here?

**Pull request merged**